common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoFrameToContributionResponseConverter.java

```diff
@@ -0,0 +1,90 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+import ai.h2o.mojos.deploy.common.rest.model.ContributionOutputGroup;
+import ai.h2o.mojos.deploy.common.rest.model.ContributionResponse;
+import ai.h2o.mojos.deploy.common.rest.model.Row;
+import ai.h2o.mojos.runtime.frame.MojoFrame;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Converts the resulting predicted {@link MojoFrame} into the API response object {@link
+ * ContributionResponse}.
+ */
+public class MojoFrameToContributionResponseConverter
+        implements Function<MojoFrame, ContributionResponse> {
```

Comment:
QQ: If something (Spring Framework?) is leveraging the fact that this class implements `Function`, is it affected by the fact that the implemented `apply()` method is only for classification models?

If this class isn't bound to implementing `Function`, then naming the two methods something descriptive, instead of being overloaded methods, would help follow what they're each used for.